powersimdata/input/converter/tests/test_pypsa_to_profiles.py

```diff
@@ -0,0 +1,35 @@
+from importlib.util import find_spec
+
+import pytest
+
+from powersimdata.input.converter.pypsa_to_grid import FromPyPSA
+from powersimdata.input.converter.pypsa_to_profiles import (
+    get_pypsa_demand_profile,
+    get_pypsa_gen_profile,
+)
+
+
+@pytest.mark.skipif(find_spec("pypsa") is None, reason="Package PyPSA not available.")
+def test_extract_solar():
+    import pypsa
+
+    n = pypsa.examples.ac_dc_meshed()
+    profile = get_pypsa_gen_profile(n, "solar")
```

Comment:
I'm getting an empty data frame for `profile` when I run this, is that expected?

Reply:
My mistake, `solar` is not available in the `Network` object
```
>>> import pypsa
>>> n = pypsa.examples.ac_dc_meshed()
>>> n.generators.carrier.unique()
array(['wind', 'gas'], dtype=object)
```

Reply:
It works for `wind`:
```
>>> from powersimdata.input.converter.pypsa_to_profiles import  get_pypsa_gen_profile,
>>> profile = get_pypsa_gen_profile(n, "wind")
>>> profile
                     Manchester Wind  Norway Wind  Frankfurt Wind
UTC                                                              
2015-01-01 00:00:00        74.401590    97.458320       61.498624
2015-01-01 01:00:00        38.859806    48.129038       82.820141
2015-01-01 02:00:00        18.695339    40.722580       13.581160
2015-01-01 03:00:00        20.608338    59.996496      106.334432
2015-01-01 04:00:00        50.152446    52.446822       94.490858
2015-01-01 05:00:00        48.287873     0.969271       57.876917
2015-01-01 06:00:00        54.312604    22.045336        8.568231
2015-01-01 07:00:00        28.904209    82.391850        6.492582
2015-01-01 08:00:00        49.728324    55.622973       27.340994
2015-01-01 09:00:00        41.721470    43.941604       11.886619
```

Reply:
I will change the test for `wind`

Reply:
Done